rules/macos/defense_evasion_safari_config_change.toml

```
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2021/01/14"
+maturity = "production"
+updated_date = "2021/01/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies changes to Safari configuration using the built-in defaults command. Adversaries may attempt to enable or
+disable certain Safari settings such as enabling JavaScript from Apple Events or equivalent to ease users browsing
+hijack.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Safari Settings Modification using Defaults Command"
+references = ["https://objectivebythesea.com/v2/talks/OBTS_v2_Zohar.pdf"]
+risk_score = 43
+rule_id = "6482255d-f468-45ea-a5b3-d3a7de1331ae"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Defense Evasion"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
```

```suggestion
event.category:process and event.type:start and
```
`process_started` isn't used anymore for `auditbeat`